packages/core/src/base/cacheManagerKeyv.ts

```diff
@@ -179,7 +179,9 @@ export class CacheManagerKeyv implements ICacheManager {
     ) {
       return;
     }
-
+    sdkLogger.debug(
+      `Request found in cache - metadata key: ${metadataKey}, content key: ${contentKey}`
```

Comment:
It's actually the response that's cached, not the request

Reply:
Since the key is request, I called it Request. But I can see how this is confusing.  How about making it generic something like "Cache entry found - ..."?